OloEngine/src/OloEngine/Core/InputRebindController.cpp

```
@@ -0,0 +1,288 @@
+
+        if (removeFromConflicts)
+        {
+            // Erase the binding from every OTHER action so it maps to exactly one action.
+            for (auto& [name, act] : map.Actions)
+            {
+                if (name == actionName)
+                {
+                    continue;
+                }
+                const auto it = std::ranges::find(act.Bindings, binding);
+                if (it != act.Bindings.end())
```

Cppcheck (2.21.0)

[style] The function 'SetTargetContext' is never used.

(unusedFunction)